package.json

```diff
@@ -16,7 +16,7 @@
         "npm": ">=5.4"
     },
     "dependencies": {
-        "elasticio-sailor-nodejs": "2.2.0",
+        "elasticio-sailor-nodejs": "2.2.1",
```

Comment:
2.2.x?

Reply:
See @drobiazko 's thoughts on why that would be a bad idea: https://elasticio.slack.com/archives/G6DRG6HEW/p1507124793000172